File: v08/zad1-2/Osoba.cpp
#include "Osoba.hpp"

Osoba::Osoba(char *ime, char *prezime) : ime(ime), prezime(prezime) {}

Osoba::Osoba(const Osoba& o) : ime(o.ime), prezime(o.prezime) {}

DinString Osoba::getIme() const {
    return ime;
}

DinString Osoba::getPrezime() const {
    return prezime;
}

void Osoba::setIme(const DinString& ime) {
    this->ime = ime;
}

void Osoba::setPrezime(const DinString& prezime) {
    this->prezime = prezime;
}

void Osoba::ispis() const {
    cout << ime << " " << prezime << endl;
}

File: v11/zad2/Osoba.java
package zad2;

import java.io.Serializable;

public class Osoba implements Serializable {
	
	// Polja
	private String ime = "";
	private String prezime = "";
	
	public Osoba(String ime, String prezime) {
		super();
		this.ime = ime;
		this.prezime = prezime;
	}

	public String getIme() {
		return ime;
	}

	public void setIme(String ime) {
		this.ime = ime;
	}

	public String getPrezime() {
		return prezime;
	}

	public void setPrezime(String prezime) {
		this.prezime = prezime;
	}

	@Override
	public String toString() {
		return "Osoba [ime=" + ime + ", prezime=" + prezime + "]";
	}
	
}
